packaging/selinux/flightctl_agent.te

```diff
@@ -29,6 +32,10 @@ gen_require(`
     attribute domain;
 ')
 
+init_daemon_domain(flightctl_agent_t, flightctl_agent_exec_t)
```

Comment:
These were just moved after the gen_require to make them consistent with the other rules.